packages/components/src/tools-panel/styles.js

```diff
@@ -61,6 +78,18 @@ export const ToolsPanelItem = css`
 		margin-bottom: 0;
 		max-width: 100%;
 	}
+
+	& > .components-base-control:last-child {
```

Comment:
I needed the class, rather than element, selector here to counter the specificity of [some styles applied](https://github.com/WordPress/gutenberg/blob/22c11033b6b8bb5c60d9d84979bd97e5b5d22548/packages/block-editor/src/components/block-inspector/style.scss#L13) to base control. Could this be iterated on when we get to cleaning up the hardcoded class names in the follow-up PR to this one?

My guess is that eventually the base control component will be hooked up to the context system and we should be able to remove those styles based on it being in a tools panel. In turn, removing the need for any resetting of style here.

Reply:
Makes sense, I've updated #35058 to include `.components-base-control` too